src/backend/database/migrations/tenant/2023_11_16_220432_update_person_id_column_in_devices_table.php

```diff
@@ -12,7 +12,7 @@
      */
     public function up() {
         Schema::connection('tenant')->table('devices', function (Blueprint $table) {
-            $table->integer('person_id')->nullable()->change();
```

Comment:
Why are we adding `unsigned`? I didn't doesn't seem to be there before.

Reply:
Yeah, I understand. But it looks like this `unsigned` constraint wasn't there before. But might be I'm missing something. Did we maybe remove this constraint in a later migration?